lms/static/sass/shared/_fields.scss

```diff
@@ -0,0 +1,103 @@
+// lms - shared - fields
+// ====================
+
+
+.u-field {
+    padding: $baseline 0;
+    border-bottom: 1px solid $gray-l5;
+    border: 1px dashed transparent;
```

Comment:
Some of these rules apply only to the profile view so I think they should move into that sass file. Here is the edited set of rules: 

shared/_fields.scss: 
```
.u-field {
  padding: $baseline 0;
  border-bottom: 1px solid $gray-l5;
}
```

views/_learner-profile.scss: 
```
.u-field {
  border: 2px dashed transparent;
  border-radius: 3px;

  &.mode-placeholder:hover {
    border: 2px dashed $link-color;
    }  

  &.editable-toggle.mode-display:hover {
    background-color: $m-blue-l4;
    border-radius: 3px;

    .message-can-edit {
      display: inline-block;
      color: $link-color;
    }
  }
}

```